shell/platform/windows/flutter_window.cc

```diff
@@ -277,4 +281,22 @@ PointerLocation FlutterWindow::GetPrimaryPointerLocation() {
   return {(size_t)point.x, (size_t)point.y};
 }
 
+void FlutterWindow::OnThemeChange() {
+  HIGHCONTRAST high_contrast = {.cbSize = sizeof(HIGHCONTRAST)};
+  // API call is only supported on Windows 8+
+  if (SystemParametersInfoW(SPI_GETHIGHCONTRAST, sizeof(HIGHCONTRAST),
+                            &high_contrast, 0)) {
+    BOOL hc_on = high_contrast.dwFlags & HCF_HIGHCONTRASTON;
+    // Currently, only FlutterWindowsView should be used as delegate on Windows
+    binding_handler_delegate_->UpdateHighContrastEnabled(hc_on);
+  } else {
+    FML_LOG(INFO) << "Failed to get status of high contrast feature,"
+                  << "support only for Windows 8 + ";
+  }
+}
+
+void FlutterWindow::UpdateInitialAccessibilityFeatures() {
```

Comment:
"UpdateInitial" seems a bit strange. What do you think of renaming this `SendAccessibilityFeatures` and then making `OnThemeChange` call `SendAccessibilityFeatures`?

```suggestion
  virtual void SendInitialAccessibilityFeatures() = 0;
```